### PR TITLE
add reference to kartoza/kobocat-template

### PR DESCRIPTION
**Pull request by boney-bun**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -7,3 +7,7 @@
 [submodule "src/kobocat"]
 	path = src/kobocat
 	url = https://github.com/kobotoolbox/kobocat.git
+[submodule "src/kobocat-template"]
```
Comment (lucernae) [inline .gitmodules:10]: What repo is this @boney-bun ?

**Comment by lucernae** [comment]
Please make changes to develop first, btw.

File: .gitmodules (hunk 1 above)
Comment (boney-bun) [reply .gitmodules:10]: it's a new repo i've created for the kobocat's style.
originally, the style is generated from https://github.com/kobotoolbox/kobocat-template. 
we need to override the repo, so i added the repo.

**Comment by boney-bun** [comment]
i pushed this PR to a new `develop` branch (there was no `develop` branch before).

**Comment by lucernae** [comment]
This PR diverges from the original intention. I will close it.
The changes should be on develop branch first.

**Pull request closed without merging**